plugins/webhooks/github.js

```diff
@@ -30,14 +30,15 @@ function stopRunningBuild(build) {
  * @param  {String}       options.username      User who created the PR
  * @param  {String}       options.prRef         Reference to pull request
  * @param  {Pipeline}     options.pipeline      Pipeline model for the pr
- * @param  {Hapi.request} request Request from user
- * @param  {Hapi.reply}   reply   Reply to user
+ * @param  {Hapi.request} request               Request from user
+ * @return {Promise}
  */
-function pullRequestOpened(options, request, reply) {
+function runMainJob(options, request) {
```

Comment:
I think the name of this function is a little unfortunately confusing, since we actually require a `main` job in every project, and this isn't actually running that job. `startPRJob` might be more accurate.